pkg/pillar/utils/globalutils.go

```diff
@@ -6,12 +6,11 @@
 package utils
 
 import (
-	"os"
-
 	"github.com/lf-edge/eve/pkg/pillar/base"
 	"github.com/lf-edge/eve/pkg/pillar/pubsub"
 	pubsublegacy "github.com/lf-edge/eve/pkg/pillar/pubsub/legacy"
 	"github.com/lf-edge/eve/pkg/pillar/types"
+	"os"
```

Comment:
We are gradually moving in the other directioj to have the standard packages like "os" up front followed by the github.com etc packages. So please revert this change.

Reply:
Done.